frontend/src/util/fee-estimator.js

```diff
@@ -6,6 +6,15 @@ let baseFee = '100',
     lastFetched = 0,
     callPromise
 
+export const confidenceValues = ['Low', 'Normal', 'High']
+
+export function resolveConfidenceFee(value = 'Normal') {
+    const index = confidenceValues.findIndex(v => v === value)
+    if (index === -1) //Didn't find available value
+        return false
+    return baseFee + baseFee * index //resolve calculated value
+}
```

Comment:
That's completely wrong logic...

Reply:
Yeah, you said "I'll give you a formula for this", and after that I can apply it here, but for now I'm only using this "wrong logic" to see the value changes for testing. 
Tell me in detail what formula to calculate the values ​​and I will rewrite the function, in fact, only replace the logic here and everything will be in its place